File: java/client/test/com/thoughtworks/selenium/corebased/TestComments.java
package com.thoughtworks.selenium.corebased;

import com.thoughtworks.selenium.InternalSelenseTestBase;
import org.junit.Test;

public class TestComments extends InternalSelenseTestBase {
	@Test public void testComments() throws Exception {
		selenium.open("../tests/html/test_verifications.html?foo=bar");
		verifyTrue(selenium.getLocation().matches("^[\\s\\S]*/tests/html/test_verifications\\.html[\\s\\S]*$"));
		verifyEquals(selenium.getValue("theText"), "the text value");
		verifyEquals(selenium.getValue("theHidden"), "the hidden value");
		verifyEquals(selenium.getText("theSpan"), "this is the span");
	}
}

File: dotnet/test/webdriverbackedselenium.tests/TestComments.cs
using NUnit.Framework;
using System.Text.RegularExpressions;

namespace Selenium.Tests
{
    [TestFixture]
    public class TestComments : SeleniumTestCaseBase
    {
        [Test]
        public void Comments()
        {
            selenium.Open("../tests/html/test_verifications.html?foo=bar");
            Assert.IsTrue(Regex.IsMatch(selenium.GetLocation(), "^[\\s\\S]*/tests/html/test_verifications\\.html[\\s\\S]*$"));
            Assert.AreEqual(selenium.GetValue("theText"), "the text value");
            Assert.AreEqual(selenium.GetValue("theHidden"), "the hidden value");
            string txt = selenium.GetText("theSpan");
            Assert.AreEqual(selenium.GetText("theSpan"), "this is the span");
        }
    }
}
